apis/python/test/test_cloud.py

```diff
@@ -67,6 +69,31 @@ def test_cloud_flat(self):
         _, result_i = index.query(queries, k=k)
         assert accuracy(result_i, gt_i) > MINIMUM_ACCURACY
 
+    def test_cloud_vamana(self):
+        source_uri = "tiledb://TileDB-Inc/sift_10k"
+        queries_uri = siftsmall_query_file
+        gt_uri = siftsmall_groundtruth_file
+        index_uri = CloudTests.vamana_index_uri
+        k = 100
+        nqueries = 100
+
+        load_fvecs(queries_uri)
+        gt_i, gt_d = get_groundtruth_ivec(gt_uri, k=k, nqueries=nqueries)
+
+        vs.ingest(
+            index_type="VAMANA",
+            index_uri=index_uri,
+            source_uri=source_uri,
+            input_vectors_per_work_item=5000,
+            config=tiledb.cloud.Config().dict(),
+            # TODO(paris): Fix and then change to Mode.BATCH.
+            mode=Mode.LOCAL,
+        )
+
+        # TODO(paris): Fix error from loading this URI and then re-enable, and add the rest of the test.
```

Comment:
Do we have a story for this? We should also test a simple query for completeness.